docs/ref-manual/cvl/invariants.md

```diff
@@ -154,15 +155,17 @@ commands are also possible.
 
 Preserved blocks are listed after the invariant expression (and after the filter
 block, if any), inside a set of curly braces (`{ ... }`).  Each preserved block
-consists of the keyword `preserved` followed by an optional method signature,
-an optional `with` declaration, and finally the block of commands to execute.
+consists of the keyword `preserved` followed by an optional method signature 
+(or `fallback()`), an optional `with` declaration, and finally the block of commands 
+to execute.
 
 If a preserved block specifies a method signature, the signature should
```

Comment:
... should either be `fallback` or should match ...

Reply:
done